dali/operators/reader/loader/loader_test.cc

```diff
@@ -132,8 +136,8 @@ TYPED_TEST(DataLoadStoreTest, CocoLoaderMmmap) {
                       .AddArg("dont_use_mmap", dont_use_mmap);
     shared_ptr<dali::CocoLoader> reader(
         new CocoLoader(
-            coco_spec, offsets, boxes, labels, counts, masks_meta,
-            masks_coords, false, false, original_ids));
+            coco_spec, widths, heights, offsets, boxes, labels, counts, masks_meta,
+            masks_coords, masks_rles, masks_rles_idx, false, false, original_ids));
```

Comment:
```suggestion
            masks_coords, masks_rles, masks_rles_idx, false, false, false, original_ids));
```

There is one more boolean argument missing here unfortunately.